codegen/Templates/python/__init__api.mustache

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+# -----------------------------------------------------------------------------
+# <copyright company="Aspose" file="{{classname}}.py">
```

Comment:
@s-zubov Please, check the possibility to use {{classVarName}} here and in other templates to avoid further external modifications.

Reply:
Neither {{classVarName}} or {{classFilename}} are valid in this context. I haven't found any suitable variable in templates here, only post-processing the generated file could help.